### PR TITLE
Make skopt choose between only n_epochs thresholds

### PR DESCRIPTION
**Pull request by jasmainak**

### CODE REVIEW

**Comment by agramfort** [comment]
faster?

**Comment by jasmainak** [comment]
hmm ... that's weird, it's not really faster

File: autoreject/autoreject.py [hunk 1]
```diff
@@ -294,10 +294,13 @@ def _compute_thresh(this_data, thresh_range, method='bayesian_optimization',
         from skopt import gp_minimize
         from sklearn.cross_validation import cross_val_score
 
-        def objective(thresh):
+        threshes = np.ptp(this_data, axis=1)
+
+        def objective(n_epochs):
+            thresh = np.sort(threshes)[n_epochs]
```
Comment (jasmainak) [inline autoreject/autoreject.py]: maybe this is making it slower, let me move the `sort` outside the `objective` ...

**Comment by jasmainak** [comment]
okay I'm lost why this is still slower ...

**Comment by jasmainak** [comment]
This is not relevant anymore. We don't have `skopt` ...

**Pull request closed without merging**